Rocket.Chat/Controllers/Chat/MessagesViewModel.swift

```diff
@@ -291,7 +301,7 @@ final class MessagesViewModel {
         guard let subscription = subscription?.validated() else { return }
         guard let subscriptionDetached = subscription.unmanaged else { return }
 
-        let messages = subscription.fetchMessages(50, lastMessageDate: oldestMessage)
+        let messages = subscription.fetchMessages(30, lastMessageDate: oldestMessage)
```

Comment:
The PR is looking really good! I can feel the improvement when opening chats, but it seems to have a little inconsistency. 

On the description you said that it is fetching a page of `40` messages, and here on the code I see `30`, but the real issue is that when debbuging each time `fetchMessages` gets called I see 60 sections/messages being inserted on our list, so it seems to be doubling up this page size right here.

Reply:
@filipealva Just fixed the description of the PR to this number... where do you see 60? Can you point to the breakpoint you're using?

Reply:
@rafaelks Initially I added the breakpoint inside `RocketChatViewController`'s UIKit extension, inside the batch updates on line 45, then I print the `changeset.sectionInserted` property and it gives me `60` items, i.e. after doing the diffing there are 60 insertions. 

Now I went back on it and tried to spot the same before the diffing, and I could manage to see the method `func handle(insertions: [Int], on messagesQuery: Results<Message>)` handling `60` items regularly. 

It doesn't happens always, sometimes there are 20 insertions and another aleatory numbers, but if you keep testing you'll see that the most common number of insertions we have is `60` which is the double of the size we should be handling.

See the screenshot below:

![screen shot 2018-10-15 at 16 12 28](https://user-images.githubusercontent.com/6117280/46972778-baf31f00-d095-11e8-97ea-b7f8b07cec71.png)

Reply:
@filipealva I see... you're seeing the observer from Realm, so new results in database would result in all these new values, because the request we're doing it for 60 new messages. Good finding, I'll change the parameter to 30 for now. 👍